packages/orbit-components/src/InputFile/InputFile.stories.tsx

```diff
@@ -1,208 +1,111 @@
 import { SPACINGS_AFTER } from "../common/consts";
 
 import InputFile from ".";
 
-export default {
+const meta: Meta<typeof InputFile> = {
   title: "InputFile",
-};
-
-export const Default = ({ label }) => {
-  return (
-    <InputFile
-      label={label}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
+  component: InputFile,
 
-Default.story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["onChange", "onFocus", "onBlur", "onRemoveFile"],
+    },
   },
-};
-
-Default.args = {
-  label: "Label",
-};
-
-export const FilledWithFile = ({ label, fileName }) => {
-  return (
-    <InputFile
-      label={label}
-      fileName={fileName}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
-
-FilledWithFile.story = {
-  name: "Filled with file",
 
-  parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+  args: {
+    label: "Label",
+    onChange: action("onChange"),
+    onFocus: action("onFocus"),
+    onBlur: action("onBlur"),
+    onRemoveFile: action("removeFile"),
   },
 };
 
-FilledWithFile.args = {
-  label: "Label",
-  fileName: "file.png",
-};
+export default meta;
+type Story = StoryObj<typeof InputFile>;
 
-export const WithHelp = ({ label }) => {
-  return (
-    <InputFile
-      label={label}
-      help={
-        <div>
-          Supported files: <strong>PNG, JPG, PDF</strong>
-        </div>
-      }
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
-
-WithHelp.story = {
-  name: "With help",
+export const Default: Story = {};
 
-  parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+export const FilledWithFile: Story = {
+  args: {
+    fileName: "file.png",
   },
 };
 
-WithHelp.args = {
-  label: "Label",
-};
-
-export const WithError = ({ label, error }) => {
-  return (
-    <InputFile
-      label={label}
-      error={error}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      onRemoveFile={action("removeFile")}
-    />
-  );
-};
-
-WithError.story = {
-  name: "With error",
-
+export const WithHelp: Story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["onChange", "onFocus", "onBlur", "onRemoveFile", "help"],
```

Comment:
It's just about consistency, but for `Error` story I can see that the input field in SB is available, here it's excluded.
I can see that default values for `help` is ReactNode type, for `error` it's string type, although both of the props are `ReactNode` type.
I would align it to make consincency for both `Error` and `Help` stories. :D

Reply:
and also, in Playground story there is Help available.

Reply:
Yes, there was `help` defined as ReactNode type so I left it there as such, but if it's being changed via controls, it breaks the whole StoryBook 🤕 so I disabled the control as it's currently in production. To be consistent, I can add `error` as ReactNode to Error story and disable control as well and in the Playground both will be string and enabled.